test/AssertTest.php

```diff
@@ -1,20 +1,25 @@
 <?php
-class AssertTest extends PHPUnit_Framework_TestCase
+
```

Comment:
```suggestion
declare(strict_types = 1);

```
(here and in all other php scripts, see https://www.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration.strict for details).

Reply:
Agreed. Found and filed #154. I'll see whether all found warnings can be quickly fixed. If so, I'll fix, otherwise I'll postpone it and file an issue.

Reply:
Resolved #154 and a testing problem. Will update the branch today.